src/components/WeeklySchedule.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from "prop-types";
+import WeeklyShow from "./WeeklyShow";
+import { getDayInt } from "../lib/globals.js";
+import styles from "../styles/WeeklyShow.module.css";
+
+export default function WeeklySchedule({ shows }){
+  const showsArr = [];
+  const days = ["","Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
+
+  const firstRow = days.map((d, i) => {
```

Comment:
I think the `<th>` tag will make the table headers a bit simpler here.

Reply:
Hm, the way it is now, we set up a whole table as an array of either WeeklyShows or <divs> and then format it as a table with <th>s in the table. This made the formatting a bunch easier because we could set it all manually. We also can't switch the div for <th> because those can't exist within <td>. Unless... we could go about having a whole conditional thing were if its the first row then we format it as a <th> rather than a <td> but I think that would be more lines of code?

Reply:
Yeah, I was thinking you could almost do it manually, but this works fine!